homotopy-web/src/app/sidebar.rs

```diff
@@ -156,9 +156,27 @@ impl Component for Sidebar {
 impl Sidebar {
     fn drawer(&self, ctx: &Context<Self>) -> Html {
         let dispatch = &ctx.props().dispatch;
-        let attach_options = ctx
-            .props()
-            .proof
+        let proof = &ctx.props().proof;
+
+        if proof.show_image_export {
+            return html! {
+                <SidebarDrawer
+                    // yh: Re-using the attach style class here
+                    class="attach"
```

Comment:
Hmm could we rename this to something other than attach. Maybe "dialog" because these things behave like dialogs?